docs/content/gameobject/world-commands.md

```diff
@@ -32,145 +32,78 @@ The commands documentation is:
-  * `object context`: Optional. An arbitrary object you can associate with the command. You get this object back along with the response. This is useful when handling the response based on the information contained in this object.
-
-Returns: the request Id for this command as a `long`.
-
-The corresponding response callback is `WorldCommandResponseHandler.OnReserveEntityIdsResponse`:
-
-```csharp
-event Action<ReserveEntityIds.ReceivedResponse> OnReserveEntityIdsResponse
-```
-Callback parameters:
-  * `ReceivedResponse`: The data associated with the incoming response stored inside a [`ReceivedResponse`]({{urlRoot}}/content/world-component-commands-requests-responses) struct.
-
-The `ReserveEntityIds.ReceivedResponse` struct contains these additional properties:
-
-| Name          	| Type 	| Description                                                                                                                                                            	|
-|-------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| FirstEntityId 	| `EntityId` | If successful, an ID which is the first in a contiguous range of newly allocated entity IDs. These are guaranteed to be unused in the current deployment. Otherwise, null. |
-| NumberOfEntityIds | `int`  	| If successful, the number of IDs reserved in the contiguous range, otherwise 0.                                                                                        	|
+  * `WorldCommands.ReserveEntityIds.Request request`: The command request payload.
+  * `Action<WorldCommands.ReserveEntityIds.ReceivedResponse> callback`: Optional. A callback that will be called when the command response is received.
 
 ### DeleteEntity
-You can use the `DeleteEntity` method to request the deletion of a SpatialOS entity given its SpatialOS entity ID. It has the following signature:
 
-```csharp
-long DeleteEntity(EntityId entityId, uint? timeoutMillis = null, object context = null)
-```
-
-Parameters:
-
-  * `EntityId entityId`: The ID of the SpatialOS entity that you want to delete.
-  * `uint timeoutMillis`: Optional. Specifies the amount of time in milliseconds to wait before the command fails with a timeout status. If not specified, the default of 5 seconds of the underlying Worker SDK is used.
-  * `object context`: Optional. An arbitrary object you can associate with the command. You get this object back along with the response. This is useful when handling the response based on the information contained in this object.
-
-Returns: the request Id for this command as a `long`.
-
-The corresponding response callback is `WorldCommandResponseHandler.OnDeleteEntityResponse`:
+You can use the `SendDeleteEntityCommand` method to request the deletion of a SpatialOS entity given its SpatialOS entity ID. It has the following signature:
```

Comment:
not completely sure, but I think we use ID and Id interchangeably. probably should do a pass to use one or the other

Reply:
I always use ID. Id seems wrong unless referring to the Freudian concept of the part of you that contains instinctive drives.